ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please try to adhere to the guidelines below as much as possible when submitting your issue.
+
+If you are submitting a bug issue:
+- [ ] Please include your platform type and number, as well as your Python, scikit-learn, numpy, and scipy versions.
```

Comment:
I was thinking of including this, but i feel like it might be a bit verbose for an issue template? But then again, I don't think there's anywhere that this would fit in the existing documentation.